task05/task05/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task05
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            double y = Value(2+Value(3+Value(4+Value(5))));
+            Console.WriteLine("Результат: " + Math.Round(y, 3));
+            Console.ReadKey();
+
+            static double Value(double x)
+            {
+                return (Math.Sqrt(x));
+
+            }
```

Comment:
Не вижу смысла создавать специальный метод, который просто вызывает уже существующий метод. могли бы просто написать
```suggestion
            double y = Math.Sqrt(2+Math.Sqrt(3+Math.Sqrt(4+Math.Sqrt(5))));
            Console.WriteLine("Результат: " + Math.Round(y, 3));
            Console.ReadKey();
```
Выделить в метод повторяющееся действие можно было, например, так
 ```suggestion
            double y = Value(2, Value(3, Value(4, 5)));
            Console.WriteLine("Результат: " + Math.Round(y, 3));
            Console.ReadKey();

            static double Value(double x, double y)
            {
                return Math.Sqrt(x + Math.Sqrt(y));
            }
```